src/components/DateTime/RelativeDateTime.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import timeToString from './timeToString';
+import useDateFNSLocale from './useDateFNSLocale';
+import InvalidDate from './InvalidDate.jsx';
+
+// Component that displays the relative time and shows the absolute time on hover
+export function RelativeDateTime(props) {
```

Comment:
@queuing4oranges Not sure, this is just a thought - I think it may be better to name this file and component to `DateTimeRelative` that it has a the same name structure coming from the `DateTime`